### PR TITLE
Fixed CSS heading size issues

### PR DESCRIPTION
**Pull request by Yemaneberhan-Lemma**

### CODE REVIEW

File: layouts/_default/baseof.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+<!DOCTYPE html>
```
Comment (hussainweb) [inline layouts/_default/baseof.html:1]: Why change this? This is one of the errors reported in the prettier run.

**Review by hussainweb — CHANGES_REQUESTED** [review]

File: layouts/_default/baseof.html (hunk 1 above)
Comment (Yemaneberhan-Lemma) [reply layouts/_default/baseof.html:1]: When I run "npm run lint-fix". It changed it.

Comment (hussainweb) [reply layouts/_default/baseof.html:1]: @Yemaneberhan-Lemma, I updated npm dependencies last week. Can you run `yarn install` and then rerun `yarn lint-fix`?

Comment (Yemaneberhan-Lemma) [reply layouts/_default/baseof.html]: @hussainweb: Done.

**Pull request merged**